# joystick profiles axes and buttons #
xbox:
  twist: {joy_axis_linear: 3,
          joy_axis_angular: 2,
          scale_angular: 1,
          scale_linear: 1}

  torso: {joy_axis_updown: 7, increment: 0.01}

  arm: {rotation1_axis: 0,
        shoulder1_axis: 1,
        shoulder2_axis: 3,
        rotation2_axis: 2,
        shoulder3_up_btn: 4,
        shoulder3_down_btn: 0,
        wrist_cw_btn: 1,
        wrist_ccw_btn: 3,
        reset_arm: 14,
        inc: 0.2} #inc in rads

  gripper: {axis: 7, inc: 0.01}

  head: {right_btn: 1,
         left_btn: 3,
         up_btn: 4,
         down_btn: 0,
         inc_pan: 3,
         inc_tilt: 3}

  arm_mode_btn: 6
  safety_btn: 7

steelseries:
  twist: {joy_axis_linear: 2,
          joy_axis_angular: 3,
          scale_angular: 1,
          scale_linear: 1}

  torso: {joy_axis_updown: 7, inc_updown: 0.01}

  arm: {rotation1_axis: 0,
        shoulder1_axis: 1,
        shoulder2_axis: 3,
        rotation2_axis: 2,
        shoulder3_up_btn: 4,
        shoulder3_down_btn: 0,
        wrist_cw: 1,
        wrist_ccw: 3,
        gripper_axis: 7,
        reset_arm: 14,
        inc: 0.5} #inc in rads

  head: {right_btn: 1,
         left_btn: 3,
         up_btn: 4,
         down_btn: 0,
         inc_pan: 3,
         inc_tilt: 3}

  arm_mode: 6
  safety_btn: 7

